synapse/tests/test_lib_hiveauth.py

```diff
@@ -13,142 +14,131 @@ class AuthTest(s_test.SynTest):
 
     async def test_hive_auth(self):
 
-        with self.getTestDir() as testdirn:
-
-            async with self.getTestTeleHive() as hive:
```

Comment:
Do we want to move to deprecate the telehive interface in general?

Reply:
Yea, i'll create a ticket for that :+1: